slack-base/src/main/java/com/hubspot/slack/client/models/response/migration/MigrationExchangeResponseIF.java

```diff
@@ -0,0 +1,25 @@
+package com.hubspot.slack.client.models.response.migration;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import org.immutables.value.Value.Immutable;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+import com.hubspot.slack.client.models.response.SlackResponse;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+public interface MigrationExchangeResponseIF extends SlackResponse {
+  String getTeamId();
+
+  String getEnterpriseId();
+
+  Map<String, String> getUserIdMap();
```

Comment:
What is the format of the response in case all user IDs are invalid? Is it `"user_id_map": {}` or no property is included in the response at all?

Reply:
In case of user id map, slack always returns this map, and when all ids are invalid in slack response we get empty map.